web/src/lib/components/login/Choice.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+    import { createEventDispatcher, onMount } from "svelte";
+    import LnurlLogin from "$lib/components/login/Lnurl.svelte";
+    import NostrLogin from "$lib/components/login/Nostr.svelte";
+    import { isProduction } from "$lib/utils";
+
+    const dispatch = createEventDispatcher();
+
+    export let onLogin = () => {};
+
+    enum LoginType {
+        Lnurl,
+        Nostr,
+    }
+
+    let type = LoginType.Lnurl;
+
+    onMount(() => { type = LoginType.Lnurl });
+</script>
+
+<div class="w-full flex items-center justify-center mt-4">
+    <div class="tabs tabs-boxed">
+        <a class="tab tab-lg" class:tab-active={type === LoginType.Lnurl} href={null} on:click={() => type = LoginType.Lnurl}>LNURL</a>
+        {#if !isProduction()}
+            <a class="tab tab-lg" class:tab-active={type === LoginType.Nostr} href={null} on:click={() => type = LoginType.Nostr}>Nostr</a>
+        {/if}
+    </div>
+</div>
+
+{#if type === LoginType.Lnurl}
+    <LnurlLogin {onLogin} on:login={(_) => dispatch('login', {})} />
+{:else if type === LoginType.Nostr}
+    <NostrLogin {onLogin} on:login={(_) => dispatch('login', {})} />
+{/if}
```

Comment:
Just a UI thing, but as the Nostr component's height is so much smaller than the LNurl one, maybe you can add vertical margins/padding to the Nostr component so there isn't so much jump. Right now when you click on Nostr, the screen gets so much smaller that for some milliseconds you think "wow, it dissapeared", or "something broke".